tools/rustfmt/src/main.rs

```diff
@@ -78,7 +78,7 @@ fn edition_query(bazel_bin: &Path, edition: &str, scope: &str, current_dir: &Pat
         //             Except for targets tagged with `norustfmt`.
```

Comment:
nit: could you add no-format and no-rustfmt to this comment

Reply:
Done!